src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/CancelLink.jsx

```diff
@@ -40,17 +40,24 @@ export default function CancelLink({ appointment }) {
           });
           dispatch(startAppointmentCancel(appointment));
         }}
-        aria-label={`Cancel appointment on ${formatAppointmentDate(
-          moment.parseZone(appointment.start),
-        )}`}
+        aria-label={
+          formatAppointmentDate(moment.parseZone(appointment.start))
```

Comment:
Just a suggestion. I would move the 'moment.parseZone' to formatAppointmentDate. The caller of the function should not need to know that the date needs to be parsed first.